org.gnome.innstereo.json

```diff
@@ -0,0 +1,84 @@
+{
+    "app-id": "org.gnome.innstereo",
```

Comment:
this app-id doesn't look correct. this isn't an official GNOME project is it?

Reply:
No the project is not an official Gnome project. Should I use "com.innstereo.innstereo"?

Reply:
According to FreeDesktop's [Upstream Metadata](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-id-generic):

> The ID must follow a reverse-DNS scheme, consisting of `{tld}`.`{vendor}`.`{product}`, for example `org.kde.Gwenview` or `com.hugski.ColorHug2`.

Reply:
I moved to com.innstereo.innstereo namespace